vnext/Microsoft.ReactNative.sln

```diff
@@ -133,12 +139,12 @@ Global
 		Shared\Shared.vcxitems*{f7d32bd0-2749-483e-9a0d-1635ef7e3136}*SharedItemsImports = 4
 	EndGlobalSection
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|ARM64 = Debug|ARM64
```

Comment:
I think we purposely reorder these from the default order that VS puts them in, so that VS defaults to x64 rather than ARM64.

Reply:
I see. I will restore it then. I noticed that at some point the experience became better, but I did not realize that it was the reason.